Runtime/Scripts/GltfImport.cs

```diff
@@ -2714,6 +2721,22 @@ async Task<bool> LoadAccessorData(Root gltf)
 
             Profiler.BeginSample("LoadAccessorData.Init");
 
+            if (m_GltfRoot.scenes.Length > 0)
+            {
+                Scene.UnityMaterials materials = m_GltfRoot.scenes[0].extras.unityMaterials;
+                BufferView materialBuffer = m_GltfRoot.bufferViews[materials.bufferView];
+                byte[] materialData = GetBufferView(materials.bufferView,
+                    materialBuffer.byteOffset, materialBuffer.byteLength).ToArray();
+                string unityMaterialPath = Application.dataPath +"/UnityMaterials";
+                Directory.CreateDirectory(unityMaterialPath);
+
+                string zipPath = unityMaterialPath + "/unityMaterials.zip";
```

Comment:
Not sure if it's better to use the Temp directory for the materials.

Reply:
Can you use `Temp/` at all? If we can yeah should be better!

Reply:
OK yeah I'll test that!

Reply:
The `Temp` directory is outside the `Assets` directory, so I couldn't find and load the materials.

Instead of using the `Temp` directory, I tried deleting the materials after exiting play mode, but I'll have to do something else for the build, since there's no edit mode in the build.

Reply:
Yeah that's what I thought about using `Temp` 😅 keep it simple for now if you want, and we can add cleanup later on.

Reply:
So the difficult part is that `Application.dataPath` is used for the editor, but I think it doesn't work with the build. I'll have to rethink where I can load the assets for the build. `Resources` is used for run-time data and would be included in the build.

Reply:
Testing the build revealed a whole host of other problems - including incompatibility of some of the URP shaders with WebGL. I think I could just include this Unity material option with a feature flag, but there's more work to be done to use it in a Live Preview WebGL build.